### PR TITLE
adds missing binding for redis cluster module

### PR DESCRIPTION
**Pull request by rksinha**
- JedisCommands with READ_CLIENT_INJECTION_NAME
- Shard Supplier

@cyzhao This is dev tested with redis cluster setup and executed a workflow successfully

### CODE REVIEW

**Comment by rksinha** [comment]
@cyzhao @mudit3774 Please check

**Comment by cyzhao** [comment]
@apanicker-nflx @kishorebanala please review and merge if it looks good.

**Comment by apanicker-nflx** [comment]
Please open this PR against the dev branch.

File: server/src/main/java/com/netflix/conductor/server/RedisClusterModule.java [hunk 1]
```diff
@@ -16,5 +20,10 @@ protected void configure(){
         bind(HostSupplier.class).toProvider(ConfigurationHostSupplierProvider.class);
         bind(DynomiteConfiguration.class).to(SystemPropertiesDynomiteConfiguration.class);
         bind(JedisCommands.class).toProvider(RedisClusterJedisProvider.class);
+        bind(JedisCommands.class)
```
Comment (apanicker-nflx) [inline server/src/main/java/com/netflix/conductor/server/RedisClusterModule.java]: `JedisCommands` is bound twice, do you want to remove one?

Comment (apanicker-nflx) [reply server/src/main/java/com/netflix/conductor/server/RedisClusterModule.java]: nvm, i see that it needs a Named implementation as well

**Pull request merged**